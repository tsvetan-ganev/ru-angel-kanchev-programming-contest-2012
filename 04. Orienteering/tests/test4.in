9 4
0302Marinova w 09:00:10 09:56:16
0601Ilieva w 09:00:03 09:55:16
0104Ivanov m 08:19:46 08:53:04
0502Yanakieva w 08:31:03 09:21:25
0904Drashkova w 10:00:46 10:42:01
0704Aleksandrova w 08:57:43 09:42:02
0401Penchev m 09:17:17 09:51:01
0803Petkova w 08:54:11 09:37:28
0202Izvorova w 08:51:23 09:26:53