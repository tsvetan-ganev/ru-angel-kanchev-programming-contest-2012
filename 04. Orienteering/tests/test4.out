01 04 
02 09 08 
04 